index/templates/index/index.html

```diff
@@ -33,18 +31,18 @@ <h3 class="fs-3 fw-bold">Connecting | Participating | Supporting | Celebrating</
     <section class="container my-5">
         <div class="row">
             <div class="col">
-                <p class="fs-5">
+                <p class="fs-6">
                     Our platform is an empowering hub for the LGBTQIA+ community, providing a personalizable space to derive the best individual experience and benefit from major Pride events throughout the UK in Summer 2024. We offer information on external resources, venues, and approved safe spaces, all promoting diversity and inclusion.
                 </p>
-                <h4 class="fs-4 fw-bold">Features</h4>
+                <h5 class="fs-5 fw-bold">Features</h5>
                 <ul>
-                    <li class="fs-5"><strong>Empowering Hub:</strong> Personalize your experience to get the most out of Pride events.</li>
-                    <li class="fs-5"><strong>Resource Links:</strong> Access support groups, educational information and safe spaces.</li>
-                    <li class="fs-5"><strong>Dynamic and Supportive Community:</strong> Celebrate diversity, feel seen, heard, and valued.</li>
-                    <li class="fs-5"><strong>Real-World Impact:</strong> Our platform aims to make a positive change in individual lives.</li>
+                    <li class="fs-6"><strong>Empowering Hub:</strong> Personalize your experience to get the most out of Pride events.</li>
+                    <li class="fs-6"><strong>Resource Links:</strong> Access support groups, educational information and safe spaces.</li>
+                    <li class="fs-6"><strong>Dynamic and Supportive Community:</strong> Celebrate diversity, feel seen, heard, and valued.</li>
+                    <li class="fs-6"><strong>Real-World Impact:</strong> Our platform aims to make a positive change in individual lives.</li>
                 </ul>
-                <p class="fs-5">
-                    Join us to experience the vibrant, supportive, and dynamic environment that celebrates and empowers the LGBTQIA+ community.
+                <p class="fs-6">
+                    <strong><i>Join us to experience the vibrant, supportive, and dynamic environment that celebrates and empowers the LGBTQIA+ community.</i></strong>
```

Comment:
The `<i>` tag is outdated, and should be used only for icons such as Font Awesome. Can you convert to to `<em>` and re-send the code through?

Reply:
Will do